documentation/api/io.strimzi.api.kafka.model.KafkaConnectSpec.adoc

```diff
@@ -105,7 +105,7 @@ Kafka Connect uses the Apache `log4j` logger implementation.
 Use the `logging` property to configure loggers and logger levels.
 
 You can set the log levels by specifying the logger and level directly (inline) or use a custom (external) ConfigMap.
-If a ConfigMap is used, you set `logging.name` property to the name of the ConfigMap containing the external logging configuration. Inside the ConfigMap, the logging configuration is described using `log4j.properties`.
+If a ConfigMap is used, you set `logging.valueFrom.configMapKeyRef.name` property to the name of the ConfigMap containing the external logging configuration. Inside the ConfigMap, the logging configuration is described using `log4j.properties`. The `logging.valueFrom.configMapKeyRef.name` and `logging.valueFrom.configMapKeyRef.key` properties are mandatory. Default logging will be used if one of `name` or `key` is not set.
```

Comment:
```suggestion
If a ConfigMap is used, you set `logging.valueFrom.configMapKeyRef.name` property to the name of the ConfigMap containing the external logging configuration. Inside the ConfigMap, the logging configuration is described using `log4j.properties`. The `logging.valueFrom.configMapKeyRef.name` and `logging.valueFrom.configMapKeyRef.key` properties are mandatory. Default logging is used if the `name` or `key` is not set.
```

Reply:
Shall we write a line on what default logging means?

Reply:
What do you suggest?

Reply:
Maybe we can change to _If you do not specify a ConfigMap, default ConfigMap settings are used. If a logger is not set, default values are put into the ConfigMap. The ConfigMap is created with the custom resource when the Cluster Operator is started, then recreated after each reconciliation. The name of the ConfigMap takes the form  _CLUSTER-NAME_-_MODEL-NAME_-config, for example, my-cluster-kafka-config._  And then say you can use the name to check the settings.

Reply:
After discussing, let's go with:

_A ConfigMap using the exact logging configuration specified is created with the custom resource when the Cluster Operator is running, then recreated after each reconciliation. If you do not specify a custom ConfigMap, default logging settings are used. If a specific logger value is not set, upper-level logger settings are inherited for that logger._